packages/devtools_app/lib/src/ui/search.dart

```diff
@@ -176,7 +176,7 @@ class AutoCompleteState extends State<AutoComplete> with AutoDisposeMixin {
     // Approximation but it's pretty accurate. Could consider using a layout builder
     // or maybe build in an overlay (that's isn't visible) to compute.
     final tileEntryHeight =
-        autoComplete.autocompleteMatchTileHeight ?? box.size.height;
```

Comment:
prefer only providing one solution if possible. 
If the box.size.height is not working, we should remove it and make autoCompleteTileHeight `required` rather than having a safe solution and a broken solution.

Reply:
As mentioned above, I got rid of the `autocompleteMatchTileHeight` and am instead calculating the height to use